components/ResilienceResource/CovidBanner.module.css

```diff
@@ -4,9 +4,11 @@
 }
 
 .CovidBanner h1 {
+  font-size: 40px;
```

Comment:
@jimmyygit @LemonFace0309 is there a way to make it more responsive instead of hard coding 40px?